models/model-information.csv

```diff
@@ -1,8 +1,9 @@
\ No newline at end of file
+ransomw-model-short-rf-20220126.sav,ransomware-detection,Haim Elisha,0.1.0,This model detects ransomware from host volitile memory features collected from DOCA AppShield,Binary random forest classifier ,Training data consists of 87968 labeled AppShield processes from 32 snapshots collected from 256 unique benign and ransomware activities.,"Combined with host data from DOCA AppShield, this model can be used to detect ransomware. A training notebook is also included so that users can update the model as more labeled data is collected.",Snapshots collected from DOCA AppShield,"For each process_id and snapshot there is a probablity score between 1 and 0, where 1 is ransomware and 0 is benign.",This model was trained in the lab on windows machines,N/A,"ohen, A,. & Nissim, N. (2018). Trusted detection of ransomware in a private cloud using machine learning methods leveraging meta-features from volatile memory. In Expert Systems With Applications. (https://www.sciencedirect.com/science/article/abs/pii/S0957417418301283)",,,V100,recall= 0.9,,,946KB,N/A,,,,,,,,,,
+rootcause.onnx,rootcause-analysis,Gorkem Batmaz,0.1.0, Root cause analysis uses a binary classifier differentiating between rootcause and ordinary logs.,bert-base-uncased transformer model,Root cause analysis is a binary classifier differentiating between ordinary logs and errors/problems/root causes in the log files,This model is an example of customized transformer-based root cause analysis. It can be further fine-tuned for specific root cause analysis or predictive maintenance needs and of your enterprise using the fine-tuning scripts in the repo. The hyper parameters can be optimised to adjust to get the best results with your dataset. The aim is to get the model to predict some false positives that could be previously unknown error types.,Linux Kernel logs,Binary sequence classification as Root cause or ordinary logs,This model was trained on a very small sample subset it may not perform with other types of data without finetuning,N/A,https://arxiv.org/abs/1810.04805,1,128,V100,0.94 Higher accuracy may not always indicate success for this use case,0.96 Higher F1 may not always indicate success for this use case,0.94 Higher accuracy may not always indicate success for this use case,417MB,N/A,bert-base-uncased,128,N/A,TRUE,TRUE,11,1.7.1,3.8.10,18.04.5 LTS,4.6.1

```

Comment:
root cause model name needs to be updated here.

Reply:
changed with the latest name